__tests__/fixtures/add/install-strict-all/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "mime-types": "^2.0.0"
```

Comment:
I don't think you need this at all since you already do `yarn add left-pad` in the test

Reply:
I think I can remove `package.json` & `mirror-for-offline` folder too then?